lib/langchain/conversation.rb

```diff
@@ -59,7 +59,19 @@ def add_examples(examples)
     def message(message)
       human_message = HumanMessage.new(message)
       @memory.append_message(human_message)
-      ai_message = llm_response(human_message)
+      ai_message = llm_response
+      @memory.append_message(ai_message)
+      ai_message
+    end
+
+    # Message the model with the function name, content and return the response.
+    # @param message [String] The content or arguments for the function call
+    # @param name [String] The name of the function to be called
+    # @return [String] The response from the function call
+    def function(message, name)
```

Comment:
I think we should have this method coming from LLMs, but it is here for feedback